ui/src/hooks/useFileManage.ts

```diff
@@ -552,9 +544,8 @@ const handleFileUpload = async (
           })
         })
       );
-
     } catch (e) {
-      loadingMessage.destroy(); 
+      loadingMessage.destroy();
       onError();
     }
   }
```

Comment:
The key points that differ between the code blocks include:

1. Use of "message" instead of "globalTipsMessage"
2. Incorrect variable names, e.g. "heartBeat": typo to "handleSocketSftpData"
3. No changes in function logic

For the current date 2025-04-08, the only change made was importing Naive UI's file manage component and adding necessary dependencies. In general, there are no potential issues due to differences observed.

In conclusion, this is a maintenance pull request with minor style modifications rather than serious bugs or performance enhancements.